### PR TITLE
Add sensors to the Traccar Server documentation

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

Adds documentation for the new sensors that are being added with https://github.com/home-assistant/core/pull/111374

Note that this is a revert of home-assistant/home-assistant.io#31958 which reverted https://github.com/home-assistant/home-assistant.io/pull/31624 that was merged too soon.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/111374
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/blueprints/integrations/traccar_server_device_speed_limit.yaml [hunk 1]
```diff
@@ -7,17 +7,17 @@ blueprint:
   domain: automation
   author: ludeeus
   homeassistant:
-    min_version: 2024.2.0
+    min_version: 1970.1.1 # Placeholder that need replacement but the version is currently not known
```
Comment (ludeeus) [inline source/blueprints/integrations/traccar_server_device_speed_limit.yaml]: Needs a replacement

File: source/blueprints/integrations/traccar_server_device_enter_geofence.yaml [hunk 2]
```diff
@@ -8,17 +8,17 @@ blueprint:
   domain: automation
   author: ludeeus
   homeassistant:
-    min_version: 2024.2.0
+    min_version: 1970.1.1 # Placeholder that need replacement but the version is currently not known
```
Comment (ludeeus) [inline source/blueprints/integrations/traccar_server_device_enter_geofence.yaml]: Needs a replacement

**Review by c0ffeeca7 — DISMISSED** [review]

**Comment by c0ffeeca7** [comment]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/blueprints/integrations/traccar_server_device_enter_geofence.yaml (hunk 2 above)
Comment (ludeeus) [reply source/blueprints/integrations/traccar_server_device_enter_geofence.yaml]:
```suggestion
    min_version: 2024.5.0
```

File: source/blueprints/integrations/traccar_server_device_speed_limit.yaml (hunk 1 above)
Comment (ludeeus) [inline source/blueprints/integrations/traccar_server_device_speed_limit.yaml]:
```suggestion
    min_version: 2024.5.0
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @ludeeus :+1:

**Pull request merged**